app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/manage/actions.ts

```diff
@@ -66,3 +69,13 @@ export const resetThrottlingRate = AuthenticatedAction(async (session, formId: s
     return { error: "There was an error. Please try again later." } as ServerActionError;
   }
 });
+
+export const updateNotificationsInterval = AuthenticatedAction(
+  async (session, formId: string, notificationsInterval: NotificationsInterval) => {
+    Promise.all([
+      updateNotificationsSetting(formId, notificationsInterval),
```

Comment:
This should be updated to handle errors so we can show issues in the UI as needed.

Reply:
i.e. in other server actions we use 

```
   return { formRecord: { id: formRecord.id, updatedAt: formRecord.updatedAt } };
    } catch (_) {
      return { formRecord: null, error: "error" };
      ```

Reply:
Any error thrown is caught and handled in the UI. Because the thrown error has no info that's used in the UI e.g. no strings etc. I didn't see why we're throwing an error with empty custom info? But if you feel strongly about it I can continue to follow the current pattern?

Reply:
Unless I'm missing something the current doesn't return an error to the UI


https://github.com/user-attachments/assets/0788e09c-82f4-477a-aeee-ad627c357012

Vs if you wrap a try catch like this 


https://github.com/user-attachments/assets/62c0f390-e228-4c43-9a1d-1de54cfce420

Reply:
Thanks - wops typo meant that to a throw

Reply:
Wops meant that to be a throw - typo

Reply:
But now the pattern doesn't really match the other functions because it throws vs returns. But I guess it's closer than it was

Reply:
Though still redundant IMO

Reply:
>  still redundant

For example 

You need to check the response for an error prop ...

The server action returns a 200 it's the payload that matters


https://github.com/user-attachments/assets/08757e8e-1062-45b2-9c2a-7093025ce2c8

Reply:
Sorry, having a bit of a brain fart this morning. Updated the code :)

Reply:
Pushed a commit for you 

https://github.com/cds-snc/platform-forms-client/pull/5473/commits/55d8115beaee27d36f7f57f69055285566db15fd